packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -233,6 +242,13 @@ test.serial('makes usdc advance', async t => {
   });
   await eventLoopIteration();
 
+  const { purses } = oracles[0].getCurrentWalletRecord();
+  // XXX #10491 should not need to resort to string match on brand
+  t.falsy(
+    purses.find(p => `${p.brand}`.match(/FastLP/)),
+    'FastLP balance not in wallet record',
```

Comment:
Is there a way to assert that it _is_ in the wallet's bank in this test context? I might suggest at least saying 'FastLP balance should be in bank, not in wallet record', or maybe put a `want` in the proposal. Otherwise, looking at this assertion on its own might make it seem like we just don't expect fastlp tokens at all. Maybe it's not necessary at all?

Reply:
good idea... I think I can do that...

Reply:
Something like this should help with that:

```js
const queryClient = makeQueryClient(
  await useChain('agoric').getRestEndpoint(),
);
const { balance } = await queryClient.queryBalance('$ADDR', 'ufastlp')
```

Reply:
yeah; there's no rest endpoint in the bootstrap world

Reply:
Ah `getOutboundMessages` didn't know about this, neat.